v2 volume: add "Selective V2 Data Engine Activation" doc (#845)
longhorn/website: content/docs/1.6.0/references/settings.md

> Default: `false`

> This allows users to activate the v2 data engine based on SPDK. Currently, it is in the preview phase and should not be utilized in a production environment. For more information, please refer to [V2 Data Engine (Preview Feature)](../../v2-data-engine).

Reviewer: ```suggestion
Setting that allows you to enable the V2 Data Engine, which is based on the Storage Performance Development Kit (SPDK). The V2 Data Engine is a preview feature and should not be used in production environments. For more information, see [V2 Data Engine (Preview Feature)](../../v2-data-engine).
```

Reviewer: The suggested setting descriptions are from [Helm Values](https://longhorn.io/docs/1.6.0/references/helm-values/).

> Default: `enabled`

> This setting allows users to enable the offline replica rebuilding for volumes using v2 data engine. For more information, please refer to [Automatic Offline Replica Rebuilding](../../v2-data-engine/features/automatic-offline-replica-rebuilding).

Reviewer: ```suggestion
Setting that allows rebuilding of offline replicas for volumes using the V2 Data Engine. For more information, see [Automatic Offline Replica Rebuilding](../../v2-data-engine/features/automatic-offline-replica-rebuilding).
```
